igvm_defs/src/lib.rs

```diff
@@ -478,13 +478,16 @@ pub struct TdxPolicy {
 
 /// This region describes VTL2.
 pub const IGVM_VHF_RELOCATABLE_REGION_IS_VTL2: u8 = 0x1;
-/// RIP for the specified VP and VTL should be adjusted by the amount this
-/// region was relocated.
-pub const IGVM_VHF_RELOCATABLE_REGION_APPLY_RIP: u8 = 0x2;
+/// The starting executable address for the specified VP and VTL should be
+/// adjusted by the amount this region was relocated (RIP on x64, PC on arm64).
+pub const IGVM_VHF_RELOCATABLE_REGION_APPLY_START_ADDRESS: u8 = 0x2;
 /// GDTR for the specified VP and VTL should be adjusted by the amount this
-/// region was relocated.
+/// region was relocated. This is supported on X64 only.
 pub const IGVM_VHF_RELOCATABLE_REGION_APPLY_GDTR: u8 = 0x4;
 
+pub const IGVM_VHF_RELOCATABLE_REGION_APPLY_RIP: u8 =
```

Comment:
Please add a backlink doc comment to `START ADDRESS` and perhaps a note about why this exists (for compat)